plugin/pkg/auth/authenticator/password/keystone/keystone.go

```diff
@@ -45,18 +45,18 @@ func (keystoneAuthenticator *KeystoneAuthenticator) AuthenticatePassword(usernam
 		glog.Info("Failed: Starting openstack authenticate client")
 		return nil, false, errors.New("Failed to authenticate")
 	}
-
+//FIXME user_id, not user_name...
 	return &user.DefaultInfo{Name: username}, true, nil
 }
 
 // NewKeystoneAuthenticator returns a password authenticator that validates credentials using openstack keystone
-func NewKeystoneAuthenticator(authURL string) (*KeystoneAuthenticator, error) {
-	if !strings.HasPrefix(authURL, "https") {
-		return nil, errors.New("Auth URL should be secure and start with https")
-	}
-	if authURL == "" {
-		return nil, errors.New("Auth URL is empty")
+func NewKeystoneAuthenticator(configPath string) (*KeystoneAuthenticator, error) {
+	if configPath != "" {
+		cfg, _, err := keystonetoken.LoadConfig(configPath)
```

Comment:
hmm.... I was hoping there could be both enabled and it would support either by the kubectl flags. but your right, it seems like only one can be enabled at a time... we may need yet another cli flag. Or get rid of username/password all together. :/